package com.stackroute.utility;

import static org.junit.Assert.*;

import org.junit.Test;

public class ExceptionCreationTest {

	@Test
	public void testFirstException() {
		ExceptionCreation ec= new ExceptionCreation();
		String expectedValue ="java.lang.NegativeArraySizeException";
		String actualValue=ec.firstException();
		assertEquals(expectedValue,actualValue);
	}

	@Test
	public void testSecondException() {
		ExceptionCreation ec= new ExceptionCreation();
		String expectedValue ="java.lang.IndexOutOfBoundsException";
		String actualValue=ec.secondException();
		assertEquals(expectedValue,actualValue);	
		}

	@Test(expected = NullPointerException.class)
	public void testThirdException() {
		ExceptionCreation ec= new ExceptionCreation();
		String expectedValue ="java.lang.IndexOutOfBoundsException";
		String actualValue=ec.secondException();
		assertEquals(expectedValue,actualValue);	
		}

}
